docs/_pages/docs/03-understand/09-swaps.md

```diff
@@ -8,6 +8,73 @@ sidebar:
 src: "_pages/docs/03-understand/09-swaps.md"
 ---
 
-<!-- TODO: explain and show (UI) how to do P2P LN<-> onchain swaps -->
+Apart from the various fiat payment methods available, there are something
+called *Swap Destinations* as well. These are payment methods, but for BTC, on a
+network other than the Lightning Network. This is usefull if you want to
+exchange lightning satoshis for on-chain satoshis (or on any other network like
+Liquid BTC if you prefer). This process of exchanging satoshis on the the
+lightning network for on-chain satoshis is usually referred to as a "swap".
 
-{% include wip %}
\ No newline at end of file
+The table below makes it simple to understand swap in terms of "buyer" and "seller":
+
+| Side   | Sends         | Recieves     | Swap type |
+|--------|---------------|--------------|-----------|
+| Seller | ⚡BTC         | 🔗 BTC       | Swap out  |
+| Buyer  | 🔗 BTC        | ⚡BTC        | Swap in   |
+
+
+### How to do a P2P Swap
+
+If you want to recieve satoshis over the Lignthing Network in exchange for your
```

Comment:
Exactly. Maybe a good sentence to drop in for absolute clarity is `"In RoboSats you always buy or sell LN Sats"`